test-apps/display-performance-test-app/package.json

```diff
@@ -59,7 +59,7 @@
     "@itwin/perf-tools": "workspace:*",
     "@itwin/build-tools": "workspace:*",
     "@itwin/eslint-plugin": "workspace:*",
-    "@bentley/react-scripts": "4.0.3",
+    "@bentley/react-scripts": "5.0.0-dev.4",
```

Comment:
Are we safe to release the 5.0.0 version of it or are we going to wait on 3.2.0 release first?

Reply:
I spoke with Bill G about this. I was thinking we can get an official stable 5.0 release for 3.3/3.4. 

Main people asking for this was Stefan for the showcase and Mike B for electron bundling.

Reply:
So we are removing this from the 3.2 release plan?

Reply:
I think this PR should still be merged today since it's us testing the 5.0.0 version but it just wouldn't be fully released.

@aruniverse if that's the case, I'll approve this PR and we can get it merged but not mention the support in NextVersion.md yet. We'll save that for 3.3.